aurora-keycloak.tf

```diff
@@ -0,0 +1,14 @@
+module "aurora_keycloak" {
```

Comment:
naming here implies that there will be aurora clusters other than the keycloak aurora cluster in this config

Reply:
updated to aurora